src/OutputFormatter/BaselineOutputFormatter.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SensioLabs\Deptrac\OutputFormatter;
+
+use SensioLabs\Deptrac\Console\Output;
+use SensioLabs\Deptrac\RulesetEngine\Context;
+use SensioLabs\Deptrac\RulesetEngine\SkippedViolation;
+use SensioLabs\Deptrac\RulesetEngine\Violation;
+use Symfony\Component\Yaml\Yaml;
+
+final class BaselineOutputFormatter implements OutputFormatterInterface
+{
+    private const DUMP_BASELINE = 'baseline-dump';
+
+    public function __construct()
```

Comment:
No need for an empty constructor. This can be removed

Reply:
yeah you are right. It's removed now :)